LEGO1/act1state.h

```diff
@@ -21,6 +21,12 @@ class Act1State : public LegoState {
 	{
 		return !strcmp(name, Act1State::ClassName()) || LegoState::IsA(name);
 	};
+
+	inline void SetUnknown18(MxU32 p) { m_unk18 = p; }
```

Comment:
Ideally we name the parameter the same as the attribute it writes to (`p_unk18`)